### PR TITLE
feat [added homepage]

### PR DESCRIPTION
**Pull request by Radheshyam1234**
Added Homepage with Categories and Brands
homepage without navigation
Homepage2
Homepage3

### CODE REVIEW

File: src/Components/Home/Slider.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { Link } from "react-router-dom";
+import { SliderImages } from "./../../db";
+
+export const Slider = () => {
+  const [sliderIndex, setSliderIndex] = useState(0);
+
+  return (
+    <>
+      <div className="slides-container">
+        <div style={{ display: "block" }} className="mySlides fade">
```
Comment (Priya-Patel31) [inline src/Components/Home/Slider.js:11]: don't use inline styling

File: src/Components/Home/Slider.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          <img
+            loading="lazy"
+            src={SliderImages[sliderIndex].img}
+            alt="Image"
+            className="img-responsive"
+          />
+        </div>
+
+        {sliderIndex !== 0 && (
+          <a
```
Comment (Priya-Patel31) [inline src/Components/Home/Slider.js:21]: use button instead of anchor tag

File: src/Components/Home/Slider.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            onClick={() => {
+              setSliderIndex(sliderIndex - 1);
+            }}
+          >
+            &#10094;
+          </a>
+        )}
+
+        {sliderIndex !== SliderImages.length - 1 && (
+          <a
```
Comment (Priya-Patel31) [inline src/Components/Home/Slider.js:32]: same as above

File: src/Components/Home/home.css [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  display: inline-block;
+}
+
+.category-title {
+  color: var(--dark-secondary-color);
+}
+
+.card-hover {
+  cursor: pointer;
+  /* box-shadow: 1px 4px 12px 0px var(--light-secondary-color); */
```
Comment (Priya-Patel31) [inline src/Components/Home/home.css:20]: remove commented code

File: src/styles.css [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+.w-30 {
+  width: 30%;
+}
+.w-70 {
+  width: 70%;
+}
+.box-shadow {
+  box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px,
```
Comment (Priya-Patel31) [inline src/styles.css:29]: use variables rather than using colors directly.

**Review by Priya-Patel31 — APPROVED** [review]
Everything seems good to me . Just have requested few changes just go through it and make those changes .

File: src/Components/Home/Home.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+        })}
+      </div>
+
+      <h3 className="text-banner-title text-bold">Explore Top Brands</h3>
+      <div className="spacer-3rem"></div>
+
+      <div className="grid-4-column-layout">
+        {Brands.map((brand) => {
+          return (
+            <div className="card-vertical card-hover box-shadow">
```
Comment (kedark152) [inline src/Components/Home/Home.js:38]: You can create and add component here and pass props

File: src/Components/Home/Home.js [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+  return (
+    <div>
+      <Slider />
+      <h3 className="text-banner-title text-bold">Categories to Bag</h3>
+      <div className="spacer-3rem"></div>
+
+      <div className="grid-4-column-layout">
+        {Categories.map((category) => {
+          return (
+            <div className="card-vertical card-hover box-shadow">
```
Comment (kedark152) [inline src/Components/Home/Home.js:17]: You can create a component here and pass props

File: src/Components/Home/home.css [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+  transition: 0.5s ease;
+}
+
+.card-hover:hover {
+  box-shadow: var(--primary-color) 0px 1px 2px 0px,
```
Comment (kedark152) [inline src/Components/Home/home.css:25]: You can use variable for box shadow

**Review by kedark152 — APPROVED** [review]
Code is written well.
 I have suggested few changes such as to use re-usable components and you can pass props into it.
 **Just a Suggestion:** Also if possible you can check out https://cloudinary.com/ website where you can upload image and then you can directly use image links.

**Pull request merged**